src/main/java/com/binance/dex/api/client/BinanceDexNodeApi.java

```diff
@@ -1,17 +1,15 @@
 package com.binance.dex.api.client;
 
 import com.binance.dex.api.client.domain.BlockMeta;
-import com.binance.dex.api.client.domain.Token;
 import com.binance.dex.api.client.domain.jsonrpc.*;
 import retrofit2.Call;
 import retrofit2.http.GET;
-import retrofit2.http.Path;
 import retrofit2.http.Query;
 
 public interface BinanceDexNodeApi {
 
-    @GET("/abci_query?path=%22/store/acc/key%22")
-    Call<JsonRpcResponse<AccountResult>> getAccount(@Query("data") String address);
+    @GET("/abci_query")
+    Call<JsonRpcResponse<AccountResult>> getAccount(@Query("path") String pathWithAddress);
```

Comment:
we may need to keep the old one, but add a new one. please sync with go-sdk,

Reply:
decide to add a new api for the origin one, but add detailed description